napari/resources/stylesheet.qss

```diff
@@ -925,3 +929,47 @@ QtDeleteShapeButton {
 QtDeletePointsButton {
    image: url(":/icons/{{ folder }}/delete_shape.svg");
 }
+
+/* ------------- QMainWindow --------- */
+/* Styling the QMainWindow is important because
+any QDockWidgets will use the MainWindow styles
+as long as they are docked (though they use the
+style of QDockWidget when undocked) */
+
+QMainWindow {
+  background: {{ background }};
+}
+
+QMainWindow::separator {
+    background: {{ foreground }};
+    width: 2px; /* when vertical */
+    height: 2px; /* when horizontal */
+}
+
+QMainWindow::separator:hover {
+    background: {{ highlight }};
+}
+
+QDockWidget::title {
+  background-color: {{ background }};
+}
+
+QDockWidget::title:hover {
+  background-color: {{ foreground }};
+}
+
+QDockWidget::close-button {
+    subcontrol-position: top left;
+    subcontrol-origin: margin;
+    position: absolute;
+    top: 0px; left: 6px; bottom: 0px;
+    width: 18px;
+}
+
+QDockWidget::float-button {
+    subcontrol-position: top left;
+    subcontrol-origin: margin;
+    position: absolute;
+    top: 0px; left: 24px; bottom: 0px;
+    width: 18px;
+}
```

Comment:
```suggestion
}

```

Reply:
@tlambert03 can you make this small change and then I think we can merge.

Reply:
@tlambert03 can you make this small change and then I think we can merge.